metadata.json

```diff
@@ -60,6 +70,6 @@
     }
   ],
   "dependencies": [
-  
+
```

Comment:
Ah weird, didn't know that ... that's ... peculiar.

Updated per your request and repushed.